### ISSUE TITLE
Use correct players endpoints

### ISSUE THREAD

**Issue by Tumne** [kammanz/raptors-app#15]
After taking sometime to explore the api endpoints, it seems like the one that is currently being used to retrieve playersList, `leaguePlayerGeneralStats`, does not show proper info for the list.

Once the PR for #8 is merged into master. Let's change the endpoint to use `teamRoster` instead.

- [x] Change endpoint for PlayersList component to teamRoster promise
- [x] Show all players photos for PlayersList
- [x] Implement old endpoint for PlayerDetails component 

Make sure its functionality sound

**Comment by Tumne** [comment]
NOTE: For loading player Details, we need to do the follow:

1. Preload whatever details the user clicks on
2. Fetch player stats details from the endpoint (I believe you only need the playerID, explore endpoints)
3. In your reducer merge both current `player roster details` and `player stats details` and return that object, as an example idea:

<img width="530" alt="Screen Shot 2020-01-21 at 7 03 43 PM" src="https://user-images.githubusercontent.com/623755/72854130-7a635780-3c81-11ea-8b8b-11b62542c34a.png">
